pysteps/io/importers.py

```diff
@@ -131,6 +133,230 @@
 except ImportError:
     PYPROJ_IMPORTED = False
 
+try:
+    import pygrib
+
+    PYGRIB_IMPORTED = True
+except ImportError:
+    PYGRIB_IMPORTED = False
+
+
+def _check_coords_range(selected_range, coordinate, full_range):
+    """Check that the coordinates range arguments follow the expected pattern in the **import_mrms** function."""
+
+    if selected_range is None:
+        return sorted(full_range)
+
+    if not isinstance(selected_range, (list, tuple)):
+
+        if len(selected_range) != 2:
+            raise ValueError(f"The {coordinate} range must be None or a two-element tuple or list")
+
+        selected_range = list(selected_range)  # Make mutable
+
+        for i in range(2):
+            if selected_range[i] is None:
+                selected_range[i] = full_range
+
+        selected_range.sort()
+
+    return tuple(selected_range)
+
+
+def import_mrms(filename, fillna=np.nan, lat_range=None, lon_range=None,
+                dtype='float32', block_size=4, **kwargs):
+    """
+    Importer for NSSL's Multi-Radar/Multi-Sensor System
+    ([MRMS](https://www.nssl.noaa.gov/projects/mrms/)) rainrate product
+    (grib format).
+
+    The rainrate values are expressed in mm/h, and the dimensions of the data
+    array are [latitude, longitude]. The first grid point (0,0) corresponds to
+    the upper left corner of the domain, while (last i, last j) denote the
+    lower right corner.
+
```

Comment:
Well done! So far in pysteps we have somewhat overlooked the dtype when importing data (but this applies to the other methods, too). Maybe we should apply the same approach you're using here to all importers to finally introduce an explicit control of it.